doc/source/train/distributed-pytorch/converting-existing-training-loop.rst

```diff
@@ -91,6 +91,68 @@ training.
             .. code-block:: python
 
                 global_batch_size = worker_batch_size * train.get_context().get_world_size()
+    
+    .. tab-item:: PyTorch Lightning
+
+        Ray Train will set up your distributed process group on each worker. You only need to 
+        provide several Ray Train utilities in your Lightning Trainer definition.
+
+        .. code-block:: diff
+
+              import pytorch_lightning as pl
+            + from ray.train.lightning import (
+            +   prepare_trainer,
+            +   RayDDPStrategy,
+            +   RayLightningEnvironment,
+            + )
+
+              def train_func_per_worker():
+                ...
+                model = MyLightningModule(...)
+                datamodule = MyLightningDataModule(...)
+                
+                trainer = pl.Trainer(
+            -     devices=[0,1,2,3],
+            -     strategy=DDPStrategy(),
+            -     plugins=[LightningEnvironment()],
+            +     devices="auto",
+            +     strategy=RayDDPStrategy(),
+            +     plugins=[RayLightningEnvironment()]
+                )
+            +   trainer = prepare_trainer(trainer)
+                
+                trainer.fit(model, datamodule=datamodule)
+    
+        We provide the following utilities:
+
+        **Step 1: Configure Distributed Strategy**
+
+        Ray Train offers several subclassed distributed strategy for Lightning. 
+        These strategy retains the same argument list as the base strategy class. 
+        Internally, it correctly configures the root device and the distributed 
+        sampler arguments.
+            
+        - :class:`~ray.train.lightning.RayDDPStrategy` 
+        - :class:`~ray.train.lightning.RayFSDPStrategy` 
+        - :class:`~ray.train.lightning.RayDeepSpeedStrategy` 
+
+        **Step 2: Configure Ray Cluster Environment Plugin**
+
+        Ray Train also provides :class:`~ray.train.lightning.RayLightningEnvironment` 
+        as a specification for Ray Cluster. This utility class configures the worker's 
+        local, global, and node rank and world size.
+
+        In addition, Ray TorchTrainer has already configured the correct 
+        ``CUDA_VISIBLE_DEVICES`` for you. One should always use all available 
+        GPUs by setting ``devices="auto"``.
+        
+        **Step 3: Prepare your Lightning Trainer**
+
+        Finally, pass your Lightning trainer into
+        :meth:`~ray.train.lightning.prepare_trainer` to validate 
+        your configurations. 
```

Comment:
```suggestion
        Finally, pass your Lightning Trainer into
        :meth:`~ray.train.lightning.prepare_trainer` to validate 
        your configurations. 
```

Also worth mentioning what happens here? Should the user expect an error or a warning if validation fails?